backends/bmv2/jsonconverter.cpp

```diff
@@ -1761,6 +1761,7 @@ void JsonConverter::addLocals() {
     json->emplace("id", nextId("headers"));
     json->emplace("header_type", scalarsName);
     json->emplace("metadata", true);
+    json->emplace("pi_omit", true);  // Don't expose scalars in PI.
```

Comment:
I am using the same scalars struct in the output for storing local variables but also for scalar fields that may have been declared in the user metadata structure.